web/app/page.js

```diff
@@ -58,5 +59,39 @@ export default function Home() {
 }
 
 function allergenTimeSortCallback(a, b) {
-  return b.time - a.time;
+  if (a.time && b.time) {
+    return a.time - b.time;
+  }
+  if (!a.time) {
+    return 1;
+  } else {
+    return -1;
+  }
+}
+
+function loadJSONfromLocal(key) {
+  if (typeof window !== "undefined") {
+    const storedTimes = JSON.parse(window.localStorage.getItem(key));
+    if (storedTimes === null) {
+      console.log("LoadJSON reading null");
```

Comment:
we should remove these if we're confident that this code is working now

Reply:
The typeof is there to make sure that this doesn't run server side. It shouldn't, due to the use client directive on the page, but it's still giving me warnings in the dev environment and when I run build. I'll remove the other though.

Reply:
i only meant the console log, sorry for the confusion!